mobile/src/screens/components/Board.tsx

```diff
@@ -0,0 +1,83 @@
+// Initial chessboard code credits go to William Candillon
+// Github: https://github.com/wcandillon
+// Source Code: https://github.com/wcandillon/can-it-be-done-in-react-native/tree/master/season4/src/Chess
+import { Chess } from "chess.js";
```

Comment:
Where is chess.js?

Reply:
This is the move validator library. It is installed as a dependency.